plugins/jira-dashboard-backend/catalog-info.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: backstage-plugin-jira-dashboard-backend
+  title: '@backstage/plugin-jira-dashboard-backend'
+  description: A Backstage backend plugin that integrates towards Jira
+spec:
+  lifecycle: experimental
+  type: backstage-backend-plugin
+  owner: maintainers
```

Comment:
Only going to leave a comment on this `catalog-info.yaml` but it would apply to the other two: we should have a clear owner for this plugin. I would also update the `CODEOWNERS` file here: https://github.com/backstage/backstage/blob/master/.github/CODEOWNERS. If that's just you to start that would be fine. I know that for other plugins they have created a group in GitHub and you can have a few people in there. I do think this should be resolved in this PR so we have a clear contact(s) for the plugin who can help support it